Etch-a-Sketch: Adds a hint to the second optional credit assignment (#27633)
TheOdinProject/curriculum: foundations/javascript_basics/project_etch_a_sketch.md

In this project, you'll be creating a pretty neat toy to flex your DOM manipulation skills. You're going to build a browser version of something between a sketchpad and an Etch-A-Sketch.

This project should *not* be easy. You'll probably have to Google frequently to find the right JavaScript methods and CSS to use – in fact, that's the point! You *can* build this using the tools that you have already learned. There are plenty of resources on the net for learning stuff that we haven't covered yet if you decide you need it. We'll walk you through the basic steps, but it will be up to you to actually implement them.

> If you get totally stuck, drop by the [chatroom](https://discord.com/channels/505093832157691914). Someone will be there to point you in the right direction.

Reviewer: ```suggestion
If you get totally stuck, drop by and ask for help in [our discord server](https://discord.gg/theodinproject). Someone will be there to point you in the right direction.
```

> 1. [Follow the instructions atop Odin's Recipes project](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/recipes#setting-up-your-projects-github-repository) to set up a Git repository for this project.

Reviewer: ```suggestion
1. Follow our instructions on [setting up your project's GitHub repository](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/recipes#setting-up-your-projects-github-repository).
```
